### DISCUSSION TITLE
Back-button behaviour change inside documents in version 2.10.5.

### DISCUSSION THREAD

**Discussion by untwerth** [gsantner/markor#1889] [Q&A]
Hi!
The navigation bar's back button does something different inside documents in fdroid version 2.10.5. than before.

My Notes folder looks something like this:

```
Notes                  (folder where i keep all the files)
├── Main.md            (main file that links to other files)
├── Main               (subfolder that contains other files)
    ├── Groceries.md
    └──  Finances.md
```

I use view mode as default and I use links inside Main.md to link to other files in subfolders. 
For example- inside Main.md file I have a link:

` [Groceries](../Notes/Main/Groceries) `

which, when clicked, sends me to the Groceries.md file.  
Once inside Groceries.md file, if I click back button on the navbar it sends me back to [Notes] folder. It used to send me back to Main.md file in version 2.10.4. I can't find any settings option to return to previous behaviour?
Why the change?

P.S. my thanks for all the work you do on this great app! :)

**Comment by harshad1** [comment]
Is this still the behavior with the newest version?

**Reply by untwerth** [reply]
Hey!
Yes, it's still the same in v2.10.6.
I'm sticking to v2.10.4 for now, but I'd like to be up to date in the future 😄

**Comment by harshad1** [comment]
I just tried this and it seems to work as expected (back takes me back to the index file, which remains in view mode)
